test/parallel/test-repl.js

```diff
@@ -328,6 +328,11 @@ function error_test() {
     // or block comment. https://github.com/nodejs/node/issues/3611
     { client: client_unix, send: 'a = 3.5e',
       expect: /^SyntaxError: Invalid or unexpected token/ },
+    // Mitigate #548 issue
```

Comment:
Nit: I'd prefer this be the full URL to the issue rather than just the issue number. (Some issues scattered throughout the code are in nodejs/node-archive (formerly joyent/node, I believe) rather than nodejs/node and using the full URL eliminates guesswork. (Not the comment 3 lines above contains a full issue URL.)